pkg/archive/pom.go

```diff
@@ -0,0 +1,78 @@
+package archive
+
+import (
+	"encoding/xml"
+	"regexp"
+)
+
+type Build struct {
```

Comment:
nit: because this is in the package "archive" and public, we need to consider whether archive.Build and archive.Project are good names for these symbols. do we want to make these private instead to avoid adding a prefix?

Reply:
> we need to consider whether archive.Build and archive.Project are good names for these symbols

they are not indeed :sweat_smile: Better name would be `archive.pom.Build` perhaps. 

> do we want to make these private instead to avoid adding a prefix?

Since these types are only used in `pom.go`, it does make sense to make them private. If we decide to make it public, we can change it to `archive.pom.Build`.

Reply:
I moved all functions and types related to pom parsing to `archive/manifest/pom.go`. Exported functions are now called `manifest.Build` which I believe is a much better name.